build/configs/rtl8730e/scripts/rlx8730e_img2_ddr.ld

```diff
@@ -0,0 +1,357 @@
+/****************************************************************************

```

Comment:
I heard almost codes are the same except RAM configuration. How about keeping common script which does not include RAM configuration and including separated RAM configuration file?
I mean that split the ldscript to common part and board specific part.

Reply:
We will modify based on your suggestion, but due to the urgency now, we would like to go with this method first.
Is that okay?